mappings/net/minecraft/unmapped/ChunkBuilder.mapping

```diff
@@ -0,0 +1 @@
+CLASS net/minecraft/unmapped/C_jnliocmw net/minecraft/unmapped/ChunkBuilder
```

Comment:
This isnt really a chunk builder. Its similar but different enough this name is misleading. I would leave this unmapped for now